src/EditorFeatures/VisualBasicTest/LineCommit/CommitWithViewTests.vb

```diff
@@ -1056,5 +1056,79 @@ End Module
                 Assert.Equal(expected.NormalizedValue, testData.Workspace.Documents.Single().TextBuffer.CurrentSnapshot.GetText())
             End Using
         End Sub
+
+        <WorkItem(3119, "https://github.com/dotnet/roslyn/issues/3119")>
+        <Fact, Trait(Traits.Feature, Traits.Features.LineCommit)>
+        Public Sub MissingThenInIf()
```

Comment:
Is there a reason this needs to be tested by the line commit tests as opposed to the direct tests for the code cleanup engine? I see there's AddMissingTokensTests.cs in the tree that might be more appropriate.